pkg/log/logger.go

```diff
@@ -0,0 +1,95 @@
+package log
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"log/slog"
+	"os"
+)
+
+var ctx context.Context
+
+const (
+	levelTrace = slog.Level(-8)
+	levelFatal = slog.Level(12)
+)
+
+var levelNames = map[slog.Leveler]string{
+	levelTrace: "TRACE",
+	levelFatal: "FATAL",
+}
+
+var LogFileName string
+var LogLevel int
```

Comment:
Hm, globals are not very nice. I would pass those configurations variable in the InitLogger func as well, or in a config struct, for example.